Doc/using/windows.rst

```diff
@@ -24,15 +21,38 @@ core interpreter and library being used by a single user. The installer is also
 able to install for all users of a single machine, and a separate ZIP file is
 available for application-local distributions.
 
-Supported Versions
-------------------
-
 As specified in :pep:`11`, a Python release only supports a Windows platform
 while Microsoft considers the platform under extended support. This means that
 Python |version| supports Windows Vista and newer. If you require Windows XP
 support then please install Python 3.4.
 
-Installation Steps
+There are a number of different installers available for Windows, each with
+certain benefits and downsides.
+
+:ref:`windows-full` contains all components and is the best option for
+developers using Python for their projects. It may require administrative
+privileges to install.
```

Comment:
"It may require administrative privileges to install." seems overly scary to me. The default install doesn't require any extra privileges (although yes, I get that the C runtimes might get installed and they may need admin privileges). Maybe just leave that sentence out?

As it stands, I think new users will read from this that the store installer is the "right" one to use, which I'm not convinced is the case (certainly not at the moment, and I'm not sure will be even once the store installer is stable).